### ISSUE TITLE
Picard HsMetrics module to show usable bases as percentages

### ISSUE THREAD

**Issue by jamigo** [ewels/MultiQC#1212]
**Description of bug:**
PCT_USABLE_BASES_ON_BAIT and PCT_USABLE_BASES_ON_TARGET columns in Picard HsMetrics are being processed as bases rather than percentages by MultiQC.

**MultiQC run details (please complete the following):**
 - Command used to run MultiQC: multiqc .
 - MultiQC Version: MultiQC v1.9
 - Operating System: Red Hat Enterprise Linux 7.5
 - Python Version: Python 3.8.1
 - Method of MultiQC installation: manual

 **Additional context**
The issue is understandable, since Picard descriptions on these columns are:
"The number of aligned, de-duped, on-bait bases out of the PF bases available."
"The number of aligned, de-duped, on-target bases out of the PF bases available."
although these columns report percentages (values from 0 to 1) rather than number of bases (as one may deduct from the PCT_ column name).
https://broadinstitute.github.io/picard/javadoc/picard/picard/analysis/directed/HsMetrics.html

**What works from me**
I've edited HsMetrics.py module as follows, in case you want to consider something similar. I've included a few lines after line 277 to detect those columns and manually change their content:
```python
            elif h.find("BASES") > -1:
                if h.startswith("PCT_USABLE"):
                    headers[h]['title'] = headers[h]['title'].replace("Pct u", "U")
                    headers[h]['modify'] = lambda x: x * 100
                    headers[h]['max'] = 100
                    headers[h]['suffix'] = '%'
                    headers[h]['shared_key'] = 'percentage'
                else:
                    headers[h]['title'] = "{} {}".format(config.base_count_prefix, headers[h]['title'])
                    headers[h]['modify'] = lambda x: x * config.base_count_multiplier
                    headers[h]['shared_key'] = 'base_count'
```

**Comment by ewels** [comment]
Great spot, thanks @jamigo!

I did a bit of extra tidying whilst I was digging around with the code, so hopefully a few of the column titles should look nicer now too.

Fixed in v1.10dev.

Phil

**Comment by rollf** [comment]
[I use the word "percent" when I talk about values between 0 and 100 and the word "fraction" for values between 0 and 1].

It's unfortunate that this metric is called `PCT_USABLE...` which suggests PerCentTage but returns a fraction [actually](https://gatk.broadinstitute.org/hc/en-us/articles/360036856051-CollectHsMetrics-Picard). I also feel that the fix in a5168f8 is problematic. While the HTML report does now include percentages, the "same" metric in the JSON still contains fractions only. From an end-user perspective, it is unclear which one is correct (only the source code tells you that and/or the raw output from Picard, which has the naming issues I mentioned above).

Now, when consuming the JSON data I could multiply by 100 to get the percentage that "the end-user" wants to see. However, I am uneasy with this. What if the MultiQC devs decide that this inconsistency is bad (because people like me raise it in comments likes this 🤷 )? Then my custom `* 100.0`-code would do the wrong thing.

EDIT: My custom code will guard against any changes this using unit tests.
